src/Raven.Client/Documents/Session/SessionTimeSeriesBase.cs

```diff
@@ -124,6 +124,44 @@ public void Delete(DateTime? from = null, DateTime? to = null)
             }
         }
 
+        public void Increment(DateTime timestamp, IEnumerable<double> values)
+        {
+            if (Session.DocumentsById.TryGetValue(DocId, out DocumentInfo documentInfo) &&
+                Session.DeletedEntities.Contains(documentInfo.Entity))
+                ThrowDocumentAlreadyDeletedInSession(DocId, Name);
+
+            var op = new TimeSeriesOperation.IncrementOperation()
+            {
+                Timestamp = timestamp.EnsureUtc(),
+                Values = values.ToArray()
```

Comment:
Here can check if they passed an array, so we avoid calling `ToArray()` again

Reply:
Done